src/admin/getting_started_checks/design_removeDefaultBannersListingDetailsPage.php

```diff
@@ -50,14 +50,14 @@ public function isComplete()
         if (geoMaster::is('auctions')) {
             $name = 'listing_auction.tpl';
             $custom = file_get_contents(geoTemplate::getFilePath(geoTemplate::MAIN_PAGE, '', $name));
-            if ((strpos($custom, '<a href="#"><img src="{external file=\'images/banners/banner1_160w.jpg\'}" alt="" /></a>') !== false)) {
+            if ((strpos($custom, '<img src="{external file=\'images/banners/sample_300x100.jpg\'}"') !== false)) {
```

Comment:
The same goes here as well, we should probably check for either the older or newer version of the banners.